salon_creator/lib/screens/login_screen.dart

```diff
@@ -0,0 +1,207 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/material.dart';
+import 'package:salon_creator/authentication/sign_in.dart';
+import 'package:salon_creator/common/color.dart';
+
+class LoginPage extends StatefulWidget {
```

Comment:
file name was changed to **login_screen** but the class name is still **LoginPage**?